newIDE/app/src/MarketingPlans/MarketingPlans.js

```diff
@@ -186,11 +199,9 @@ const MarketingPlans = ({ game }: Props) => {
     async (i18n: I18nType, marketingPlan: MarketingPlan) => {
       if (!profile || !limits) return;
 
-      const {
-        creditsAmount: packCreditsAmount,
-        id,
-        nameByLocale,
-      } = marketingPlan;
+      const { id, nameByLocale } = marketingPlan;
+      const packCreditsAmount = getMarketingPlanPrice(marketingPlan);
```

Comment:
This should be `planCreditsAmount` shouldn't it?

Reply:
yes!